recipe/meta.yaml

```
@@ -107,13 +107,14 @@ outputs:
       run_exports:
         - {{ pin_subpackage("cuda-cudart", max_pin="x") }}
     files:
-      - targets/{{ target_name }}/include/*                     # [linux]
-      - targets/{{ target_name }}/include/cooperative_groups/*  # [linux]
-      - targets/{{ target_name }}/lib/*.so                      # [linux]
-      - Library\include                                         # [win]
-      - Library\lib\cuda.lib                                    # [win]
-      - Library\lib\cudadevrt.lib                               # [win]
-      - Library\lib\cudart.lib                                  # [win]
+      - targets/{{ target_name }}/include/*                             # [linux]
+      - targets/{{ target_name }}/include/cooperative_groups/*          # [linux]
+      - targets/{{ target_name }}/include/cooperative_groups/details/*  # [linux]
```

From local testing on a similar case, this appeared to work. Though let's double check

```suggestion
      - targets/{{ target_name }}/include/**                             # [linux]
```

---

After further offline discussion/exploration, it seems just dropping the `*` fixes this issue. So would suggest doing that here instead

```suggestion
      - targets/{{ target_name }}/include                               # [linux]
```